justfile

```diff
@@ -65,3 +65,6 @@ test-features arg="":
 # Remove test-generated data
 clean-data:
     ./contrib/clean_data.sh
+
+# Run all needed checks before contributing code
+pre-contributing: lint fmt test build-release
```

Comment:
I would do this instead, as the `lint` recipe includes formatting, and `test-features` is the full check run in GitHub Actions.

```suggestion
pc: lint test-features
```